src/main/java/jenkins/plugins/office365connector/model/Action.java

```diff
@@ -0,0 +1,12 @@
+package jenkins.plugins.office365connector.model;
+
+import java.util.List;
+
+public interface Action {
+
+    void setName(String name);
+
+    void setTarget(List<String> target);
```

Comment:
`setTargets` since this takes list as a parameter

Reply:
in general i agree, but currently consistent to https://github.com/jenkinsci/office-365-connector-plugin/blob/master/src/main/java/jenkins/plugins/office365connector/model/PotentialAction.java#L57

Reply:
then I believe both should be updated to be consistent

Reply:
Done